### PR TITLE
Split java cloud docs into 3 sections

### PR DESCRIPTION
**Pull request by emmawillis**

### CODE REVIEW

File: docs/sdk/example-apps.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Welcome to the Samples & Example Apps page, which showcases example applications
 | [Flutter](https://github.com/devcyclehq/flutter-client-sdk)               | [Flutter Reference Docs](/sdk/client-side-sdks/flutter.md)                     |
 | [Go](https://github.com/DevCycleHQ/go-server-sdk)                         | [Go Reference Docs](/sdk/server-side-sdks/go.md)                               |
 | [iOS](https://github.com/devcyclehq/ios-client-sdk)                       | [iOS Reference Docs](/sdk/client-side-sdks/ios.md)                             |
-| [Java (Cloud Bucketing)](https://github.com/DevCycleHQ/java-server-sdk)   | [Java (Cloud Bucketing) Reference Docs](/sdk/server-side-sdks/java-cloud.md)   |
+| [Java (Cloud Bucketing)](https://github.com/DevCycleHQ/java-server-sdk)   | [Java (Cloud Bucketing) Reference Docs](/sdk/server-side-sdks/java-cloud)   |
```
Comment (aurorawalker) [inline docs/sdk/example-apps.md:20]: Does this actually work? I thought it needed to be changed to `/java-cloud/java-cloud.md`

File: docs/sdk/server-side-sdks/java-cloud/java-cloud.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Welcome to the DevCycle Java SDK, which interfaces with the [DevCycle Bucketing API](/bucketing-api/#tag/devcycle).
+All requests, including user data are sent to DevCycle servers to ensure the User is bucketed correctly and will receive the correct variation.
+
+The SDK is available as a package on MavenCentral. It is also open source and can be viewed on Github.
+
+[![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/java-server-sdk)](https://search.maven.org/artifact/com.devcycle/java-server-sdk)
+[![GitHub](https://img.shields.io/github/stars/devcyclehq/java-server-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/DevCycleHQ/java-server-sdk)
+
+## Requirements
```
Comment (aurorawalker) [inline docs/sdk/server-side-sdks/java-cloud/java-cloud.md:13]: I think we want requirements in the 'installation' section - I believe it is also info we want pop up in context when installing (at least, that's what I've done on some other sections I've worked on).

**Review by aurorawalker — CHANGES_REQUESTED** [review]

File: docs/sdk/example-apps.md (hunk 1 above)
Comment (emmawillis) [reply docs/sdk/example-apps.md:20]: it works! it links to the same page

File: docs/sdk/server-side-sdks/java-cloud/java-cloud.md (hunk 2 above)
Comment (emmawillis) [reply docs/sdk/server-side-sdks/java-cloud/java-cloud.md:13]: oh true that makes sense, i'll move it

File: docs/sdk/example-apps.md (hunk 1 above)
Comment (emmawillis) [reply docs/sdk/example-apps.md:20]: although maybe i should keep it consistent with the other links here

File: docs/sdk/server-side-sdks/java-cloud/java-cloud.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Java SDK for Cloud Bucketing
+---
+
+Welcome to the DevCycle Java SDK, which interfaces with the [DevCycle Bucketing API](/bucketing-api/#tag/devcycle).
+All requests, including user data are sent to DevCycle servers to ensure the User is bucketed correctly and will receive the correct variation.
+
+The SDK is available as a package on MavenCentral. It is also open source and can be viewed on Github.
+
+[![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/java-server-sdk)](https://search.maven.org/artifact/com.devcycle/java-server-sdk)
```
Comment (aurorawalker) [inline docs/sdk/server-side-sdks/java-cloud/java-cloud.md:10]: New requirement as per the slack conversations - we now want these badges at the top of _all_ the subpages for this section.

https://taplytics.slack.com/archives/C032ZGWQPCJ/p1681485912733769

File: docs/sdk/features/initialization.md [hunk 4]
```diff
@@ -51,5 +51,5 @@ If a variable is first read from the cache and has a listener for [realtime upda
 
 ### [• Java Local SDK](/sdk/server-side-sdks/java-local#installation)
 
-### [• Java Cloud SDK](/sdk/server-side-sdks/java-cloud#installation)
+### [• Java Cloud SDK](/sdk/server-side-sdks/java-cloud/java-cloud-usage#installation)
```
Comment (aurorawalker) [inline docs/sdk/features/initialization.md:54]: I think I steered you wrong on this - new requirement as per slack conversation, we now want this link to point to the 'Getting Started' page

https://taplytics.slack.com/archives/C032ZGWQPCJ/p1681418554428909

**Review by aurorawalker — CHANGES_REQUESTED** [review]

**Review by aurorawalker — APPROVED** [review]

**Pull request merged**